sources/packages/web/src/components/aest/students/assessment/AwardTable.vue

```diff
@@ -0,0 +1,121 @@
+<template>
+  <v-table class="bordered">
```

Comment:
we have bootstrap class for bordered table https://getbootstrap.com/docs/4.1/content/tables/#bordered-table

Reply:
When I apply that style to v-table it is forming a thicker border which is not looking nice and not matching figma as well.
![image](https://user-images.githubusercontent.com/54600590/191581162-96f4da98-8e5d-4b8e-a884-1eab293feb8f.png)